sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/signaling/CommunicationListener.java

```diff
@@ -0,0 +1,66 @@
+package com.azure.android.communication.chat.signaling;
+
+import com.microsoft.trouterclient.ITrouterConnectionInfo;
+import com.microsoft.trouterclient.ITrouterListener;
+import com.microsoft.trouterclient.ITrouterRequest;
+import com.microsoft.trouterclient.ITrouterResponse;
+
+import org.json.JSONObject;
+
+public class CommunicationListener implements ITrouterListener {
```

Comment:
just double checking, do we want to expose this as a public type? If the end user is not suppose to use this directly then we can either make it package private or move it to implementation package.